app/models/like.rb

```diff
@@ -0,0 +1,12 @@
+# This is the like class.
+class Like < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  belongs_to :post
+  after_save :update_likes_counter
+
+  private
+
+  def update_likes_counter
+    post.increment!(:likes_counter)
+  end
+end
```

Comment:
- [ ] Create job so far :clap: But I noticed you did not create a `like controller`  and actions that allow Users to add likes to Posts. Please make sure that user should be able to add a like to posts. the` likes controller` should have a **create actions** similar post and comment controllers :+1: